authenticator_sdk/build.gradle

```diff
@@ -79,6 +79,7 @@ dependencies {
     api 'com.squareup.retrofit2:converter-gson:2.7.1'
     api 'com.squareup.okhttp3:logging-interceptor:4.9.0'
     api 'net.danlew:android.joda:2.10.2'
+    implementation 'com.jakewharton.timber:timber:4.7.1'
```

Comment:
You have added Timber to the SDK, which is used by our clients separately for the application. Add a description of how to properly configure Timber for use in their applications.
Also increase the SDK version, create a new SDK artifact, add a mention of the new version in the Pull Request header.